corehq/apps/reports/templates/reports/form/edit_submission.html

```diff
@@ -2,7 +2,7 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 
-{% block head %}{{ block.super }}
+{% block js %}{{ block.super }}
```

Comment:
we may want to add the `include` tag to the list of incompressible tags in the test so we catch it next time: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/tests/test_compress_command.py#L17